presto-spi/src/main/java/com/facebook/presto/spi/plan/AbstractJoinNode.java

```diff
@@ -11,18 +11,30 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.facebook.presto.sql.planner.plan;
+
```

Comment:
The license is repeated twice. Please delete it.

Reply:
thanks! fixed